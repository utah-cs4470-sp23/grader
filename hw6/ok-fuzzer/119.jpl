show {(! ((- 716) > ((- 61) / (202 * 25)))), (((527 != (589 - 390)) != (if (30.53 > 80.33) then true else (! true))) == (((81.1 / 49.16) <= 38.52) || false))}

