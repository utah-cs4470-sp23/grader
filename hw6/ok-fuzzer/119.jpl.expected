(ShowCmd
 (TupleLiteralExpr
  (TupleType (BoolType) (BoolType))
  (UnopExpr
   (BoolType)
   !
   (BinopExpr
    (BoolType)
    (UnopExpr (IntType) - (IntExpr (IntType) 716))
    >
    (BinopExpr
     (IntType)
     (UnopExpr (IntType) - (IntExpr (IntType) 61))
     /
     (BinopExpr (IntType) (IntExpr (IntType) 202) * (IntExpr (IntType) 25)))))
  (BinopExpr
   (BoolType)
   (BinopExpr
    (BoolType)
    (BinopExpr
     (BoolType)
     (IntExpr (IntType) 527)
     !=
     (BinopExpr (IntType) (IntExpr (IntType) 589) - (IntExpr (IntType) 390)))
    !=
    (IfExpr
     (BoolType)
     (BinopExpr
      (BoolType)
      (FloatExpr (FloatType) 30)
      >
      (FloatExpr (FloatType) 80))
     (TrueExpr)
     (UnopExpr (BoolType) ! (TrueExpr))))
   ==
   (BinopExpr
    (BoolType)
    (BinopExpr
     (BoolType)
     (BinopExpr
      (FloatType)
      (FloatExpr (FloatType) 81)
      /
      (FloatExpr (FloatType) 49))
     <=
     (FloatExpr (FloatType) 38))
    ||
    (FalseExpr)))))
